internal/bucket/lifecycle/lifecycle.go

```diff
@@ -235,9 +235,9 @@ func (lc Lifecycle) Validate() error {
 	return nil
 }
 
-// FilterActionableRules returns the rules actions that need to be executed
-// after evaluating prefix/tag filtering
-func (lc Lifecycle) FilterActionableRules(obj ObjectOpts) []Rule {
+// FilterRulesByStatusPrefixAndTags returns the rules actions that need to be executed
+// after evaluating prefix/tags filtering
+func (lc Lifecycle) FilterRulesByStatusPrefixAndTags(obj ObjectOpts) []Rule {
```

Comment:
To keep the method name shorter yet meaningful.
```suggestion
// FilterRules returns the rules that apply to opts after filtering by object prefix and tags.
// Note: Further checks are required to evaluate the resultant lifecycle.Event. See Eval.
func (lc Lifecycle) FilterRules(obj ObjectOpts) []Rule {
```